net-im/psimedia/psimedia-9999.ebuild

```diff
@@ -0,0 +1,41 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake-utils multilib eutils git-r3
```

Comment:
Please use `cmake` instead. Neither `multilib` nor `eutils` are used in here.

Reply:
done

Reply:
still seeing bogus `multilib` and `eutils` there ;)